src/filters/NumberFormat.js

```diff
@@ -0,0 +1,3 @@
+export default function (value) {
+      return parseFloat(value).toFixed(2);
+    }
```

Comment:
Maybe this is just GitHub but it looks like the formatting here has some extra spaces.